sdk/cosmosdb/cosmos/test/public/functional/container.spec.ts

```diff
@@ -178,6 +197,35 @@ describe("Containers", function (this: Suite) {
           assert.equal(err.message, "Partition key must start with '/'");
         }
       });
+      it("Is missing leading '/' - hierarchical partitions", async function () {
+        // create database
+        const database = await getTestDatabase("container CRUD bad partition key");
+
+        // create a container
+        const badPartitionKeyDefinition = ["id", "/id2"];
```

Comment:
can we add more cases of badPartitionKeyDefinition, something like invalid keys or maybe some non-existing paths in DB